vbfml/scripts/err_analysis

```diff
@@ -443,10 +456,14 @@ def evaluate(ctx, input_dir: str, sequence_type: str) -> None:
     default="history",
     help="If specified, clarify the name of the pdf save file",
 )
-def compare_hist(ctx, input_dirs: str, number_epochs: int, save_name: str) -> None:
+def compare_hist(
+    ctx, input_dirs: str, number_epochs: int, save_name: str, full_dir: bool = False
+) -> None:
     """
     Compare the statistics of the history of different models.
     """
+    if full_dir:
+        input_dirs = glob(pjoin(input_dirs[0], f"*"))
```

Comment:
This functionality of specifying a parent directory is nice, but has it been particularly useful for you to keep it in the `main`? My worry is that if we're going to do this, we need to be sure that `input_dirs` is ONLY picking up training directories and nothing else (the current `glob` syntax doesn't guarantee that). So looking down at the code, an invalid `input_dirs` will definitely get you an `AssertionError`.

To check this, you can use `os.path.isdir(dir)`, and if this evaluates to `False`, we should ignore it and move on.

Reply:
And I think we can slightly rename this: Instead of `--full-dir`, maybe the flag could be named something like `--is-parent`, so that it represents that the input directory specified by the `-i` option is the parent of all the directories we want to take a look at.

Just one more thing though: If someone specifies `--is-parent` and specifies more than one `-i` option, that is probably invalid, right? So we can include an assertion checking that: If `--is-parent` is specified, number of paths specified by `-i` must be exactly one.

Reply:
No I don't think that it is the Most important change. At some point I had to compare 10 differents history in the same dir, so I implemented that because I was lazy to type the 10 paths. So it could still be useful but not necessary. I'm not sure I will use it again.

I can do a check of the file it considers or just remove the functionality as you think is better ?

Reply:
I see thanks, it does look like a useful feature if you have a large number of models to compare indeed (e.g. maybe doing a grid search, and every model is saved under a parent directory). So I think we can keep it, but can you rename this option to `--is-parent`, and implement the checks I was mentioning above? Let me know if you have any questions or any other suggestion about it, thanks!

Reply:
I change --is-parent and did the check of multiple -i with --is-parent.

But for the check of taking only training directories in account, I'm worried that 'os.path.isdir(dir)' isn't enough. Because if another directory (of plots for instance) it will still crash 
I'm thinking about making an assert if a history.pkl exist instead

Reply:
Yeah you're right, I think it would be great to check if you can assert that `history.pkl` exists in each directory this function is looking at.

Reply:
done 
check if it is a dir and if it contains history.pkl